README.md

```diff
@@ -43,6 +43,38 @@ pipenv run grow install
 pipenv run grow run
 ```
 
+## Development without pipenv
+
+If you are using standard virtual environments or a different virtual environment manager (like pyenv) you can follwo this instructions.
```

Comment:
Fix typos; also, could you move this to a CONTRIBUTING.md if this is meant for Grow developers (not users)?